Extdn/Sniffs/Code/NoDebuggingStatementsSniff.php

```diff
@@ -0,0 +1,62 @@
+<?php
+declare(strict_types=1);
+
+namespace Extdn\Sniffs\Code;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+
+/**
+ * Class NoDebuggingStatementsSniff
+ * @package Extdn\Sniffs\File
+ */
+class NoDebuggingStatementsSniff implements Sniff
+{
+    /**
+     * @var string
+     */
+    protected $message = 'No debugging statements should be added to production code.';
+
+    /**
+     * @var int
+     */
+    protected $severity = 8;
+
+    /**
+     * @inheritdoc
+     */
+    public function register()
+    {
+        return [T_CLASS];
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function process(File $phpcsFile, $stackPtr)
+    {
+        $tokens = $phpcsFile->getTokens();
+
+        foreach ($tokens as $token) {
+            if (!isset($token['type']) || !isset($token['content'])) {
+                continue;
+            }
+
+            if ($token['type'] == 'T_EXIT' && $token['content'] == 'die') {
```

Comment:
What if it was `exit()`, not `die()`?

Reply:
Good point! Thanks! Missed that. Fixed it: https://github.com/jissereitsma/extdn-phpcs/commit/666943d108b8533e8cbce8c16981cac83cc0f3d6